docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -378,6 +378,10 @@ To get a random MAC address from a string prefix starting with '52:54:00'::
 
 Note that if anything is wrong with the prefix string, the filter will issue an error.
 
+As of Ansible version 2.8, it's also possible to initialize the random number generator from a seed. This way, you can create random-but-idempotent MAC addresses::
```

Comment:
Since Ansible 2.8 is already cut out. This should be 2.9, other changes look good to me.

Reply:
Right, changed to 2.9. Thank you